playbooks/files/dynamic_inventory.py

```diff
@@ -0,0 +1,195 @@
+#!/usr/bin/env python
+#
+# Copyright (c) 2024 Cisco and/or its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+__copyright__ = "Copyright (c) 2024 Cisco and/or its affiliates."
+__author__ = "Allen Robel"
+
+import json
+from os import environ
+
+"""
+# Summary
+
+Dynamic inventory for DCNM Collection integration tests.
+Inventory is built from environment variables.
+
+# Usage
+
+See README.md in the top-level of this repository and define the environment
```

Comment:
Are you adding more information to our README.md?  I don't see any updates to it

Reply:
Oops.  This is a holdover from when this was a separate repository.  I'll modify to remove this reference.